Not able to start server in production mode

Hi,

i am getting one issue when i am running this command npm run start.prod
```js
{"name":"Angular Universal App","type":"app","hostname":"goku","pid":14724,"level":50,"err":{"msg":"Uncaught (in promise): Error: Cannot find module './js/bundle-home.module.js'\nError: Cannot find module './js/bundle-home.module.js'\n    at Function.Module._resolveFilename (module.js:485:15)\n    at Function.Module._load (module.js:437:25)\n    at Module.require (module.js:513:17)\n    at require (internal/module.js:11:18)\n    at s (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:1793)\n    at c (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:2842)\n    at Function.r.import (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:3747)\n    at E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:891:29\n    at new ZoneAwarePromise (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29)\n    at loadChildren (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:888:51)\n    at RouterConfigLoader.loadModuleFactory (E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1)\n    at RouterConfigLoader.load (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52)\n    at MergeMapSubscriber.project (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74)\n    at MergeMapSubscriber._tryNext (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21)\n    at MergeMapSubscriber._next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12)\n    at MergeMapSubscriber.Subscriber.next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12)","url":"/","stack":"Function.Module._resolveFilename()@module.js:485:15\nFunction.Module._load()@module.js:437:25\nModule.require()@module.js:513:17\nrequire()@internal/module.js:11:18\ns()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:1793\nc()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:2842\nFunction.r.import()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:3747\n{anonymous}()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:891:29\nnew ZoneAwarePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29\nloadChildren()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:888:51\nRouterConfigLoader.loadModuleFactory()@E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1\nRouterConfigLoader.load()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52\nMergeMapSubscriber.project()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74\nMergeMapSubscriber._tryNext()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21\nMergeMapSubscriber._next()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12\nMergeMapSubscriber.Subscriber.next()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12\nresolvePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:795:31\nresolvePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:766:17\n{anonymous}()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:844:17\nZoneDelegate.invokeTask()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:425:31"},"msg":"Uncaught (in promise): Error: Cannot find module './js/bundle-home.module.js'\nError: Cannot find module './js/bundle-home.module.js'\n    at Function.Module._resolveFilename (module.js:485:15)\n    at Function.Module._load (module.js:437:25)\n    at Module.require (module.js:513:17)\n    at require (internal/module.js:11:18)\n    at s (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:1793)\n    at c (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:2842)\n    at Function.r.import (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1345:3747)\n    at E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:891:29\n    at new ZoneAwarePromise (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29)\n    at loadChildren (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:888:51)\n    at RouterConfigLoader.loadModuleFactory (E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1)\n    at RouterConfigLoader.load (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52)\n    at MergeMapSubscriber.project (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74)\n    at MergeMapSubscriber._tryNext (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21)\n    at MergeMapSubscriber._next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12)\n    at MergeMapSubscriber.Subscriber.next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12)","time":"2017-08-28T06:53:18.369Z","v":0}
[Browsersync] Proxying: http://localhost:8083
[Browsersync] Access URLs:
 --------------------------------------
       Local: http://localhost:8000
    External: http://192.168.0.109:8000
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.0.109:3001
```

Now i am getting this issue.

{"name":"Angular Universal App","type":"app","hostname":"goku","pid":13536,"level":50,"err":{"msg":"Uncaught (in promise): Error: Cannot find module './js/bundle-home.module.js'\nError: Cannot find module './js/bundle-home.module.js'\n    at Function.Module._resolveFilename (module.js:485:15)\n    at Function.Module._load (module.js:437:25)\n    at Module.require (module.js:513:17)\n    at require (internal/module.js:11:18)\n    at s (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:1793)\n    at c (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:2842)\n    at Function.r.import (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:3747)\n    at E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:884:29\n    at new ZoneAwarePromise (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29)\n    at loadChildren (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:881:51)\n    at RouterConfigLoader.loadModuleFactory (E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1)\n    at RouterConfigLoader.load (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52)\n    at MergeMapSubscriber.project (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74)\n    at MergeMapSubscriber._tryNext (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21)\n    at MergeMapSubscriber._next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12)\n    at MergeMapSubscriber.Subscriber.next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12)","url":"/","stack":"Function.Module._resolveFilename()@module.js:485:15\nFunction.Module._load()@module.js:437:25\nModule.require()@module.js:513:17\nrequire()@internal/module.js:11:18\ns()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:1793\nc()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:2842\nFunction.r.import()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:3747\n{anonymous}()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:884:29\nnew ZoneAwarePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29\nloadChildren()@E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:881:51\nRouterConfigLoader.loadModuleFactory()@E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1\nRouterConfigLoader.load()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52\nMergeMapSubscriber.project()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74\nMergeMapSubscriber._tryNext()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21\nMergeMapSubscriber._next()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12\nMergeMapSubscriber.Subscriber.next()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12\nresolvePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:795:31\nresolvePromise()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:766:17\n{anonymous}()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:844:17\nZoneDelegate.invokeTask()@E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:425:31"},"msg":"Uncaught (in promise): Error: Cannot find module './js/bundle-home.module.js'\nError: Cannot find module './js/bundle-home.module.js'\n    at Function.Module._resolveFilename (module.js:485:15)\n    at Function.Module._load (module.js:437:25)\n    at Module.require (module.js:513:17)\n    at require (internal/module.js:11:18)\n    at s (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:1793)\n    at c (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:2842)\n    at Function.r.import (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:1351:3747)\n    at E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:884:29\n    at new ZoneAwarePromise (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\zone.js\\dist\\zone-node.js:861:29)\n    at loadChildren (E:\\dev\\fusebox-angular-universal-starter\\dist\\server.js:881:51)\n    at RouterConfigLoader.loadModuleFactory (E:\\dev\\fusebox-angular-universal-starter\\packages\\router\\src\\router_config_loader.ts:61:1)\n    at RouterConfigLoader.load (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:4057:52)\n    at MergeMapSubscriber.project (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\@angular\\router\\bundles\\router.umd.js:1835:74)\n    at MergeMapSubscriber._tryNext (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:125:21)\n    at MergeMapSubscriber._next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\operator\\mergeMap.ts:115:12)\n    at MergeMapSubscriber.Subscriber.next (E:\\dev\\fusebox-angular-universal-starter\\node_modules\\rxjs\\src\\Subscriber.ts:95:12)","time":"2017-08-29T18:02:37.639Z","v":0}

--- a/fuse.ts
+++ b/fuse.ts
@@ -40,7 +40,7 @@ const appBundleInstructions = ` !> [client/${mainEntryFileName}.ts]`;
 const baseOptions = {
   homeDir: './src',
   output: `${BUILD_CONFIG.outputDir}/$name.js`,
-  cache: true,
+  cache: !isProdBuild,
   target: 'browser',
   plugins: [
     Ng2TemplatePlugin(),

